### PR TITLE
Add C to legend

### LINKED ISSUE #206: NIT: Aggregated score plot missing "C" symbol in degrees
Not priority:

Screen Shot 2022-11-22 at 7 52 25 PM

### PR DESCRIPTION
**Pull request by MonikaFu**
Closes #206

### CODE REVIEW

**Review by jdhoffa — APPROVED** [review]
lgtm

File: R/plot_scores.R [hunk 1]
```diff
@@ -163,36 +163,36 @@ legend_scores <- function() {
   )
 
   data_labels <- tibble::tibble(
-    label = c("Portfolio", "Peers", "1.8\u00B0 aligned\nthreshold"),
+    label = c("Portfolio", "Peers", "1.8\u00B0C\naligned\nthreshold"),
```
Comment (cjyetman) [inline R/plot_scores.R:166]: intended to add a newline here?

**Review by cjyetman — APPROVED** [review]
looks fine as long as that new `\n` was intended

File: R/plot_scores.R (hunk 1 above)
Comment (MonikaFu) [reply R/plot_scores.R:166]: Yes

**Pull request merged**